src/utils.js

```diff
@@ -434,3 +434,34 @@ export function is_reference(node) {
 
 	return true;
 }
+
+/**
+ * parse tsconfig.json with typescript api
+ *
+ * @param {string} tsconfig_file
+ * @returns {{include: string[]|undefined, exclude: string[]|undefined,compilerOptions: ts.CompilerOptions}}}
```

Comment:
It looks like you've got two opening braces, but three closing braces. Also, Prettier would put each property on its own line here
```suggestion
 * @returns {{
 *   include: string[]|undefined,
 *   exclude: string[]|undefined,
 *   compilerOptions: ts.CompilerOptions
 * }}
```